### DISCUSSION TITLE
Best practice adjusting the order increment step

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#1978] [Questions & Answers] [answered]
Currently order numbers are incremented by 1. M2 offers an easy way to change this. With magento 1.9 it is not so trivial as changing something in the database, it looks like this file[1] needs to be overridden, no idea if it affects other 'sequences'.

Has someone already been using something that implements this functionality and is willing to share how to do this?

[1]
./app/code/core/Mage/Eav/Model/Entity/Increment/Numeric.php

https://magento.stackexchange.com/questions/134013/how-to-set-order-increment-id-iterator-in-magento-1

**Comment by addison74** [comment, marked as answer]
Check these resources for coding yourself

https://www.atwix.com/magento/custom-order-and-customer-numbers/
https://www.atwix.com/magento/custom-order-and-customer-numbers-2/

This extension is one of the best for dealing with custom numbers for Orders/Invoices/Shipments/Credit Memos

https://amasty.com/custom-order-number.html

**Reply by f1-outsourcing** [reply]
Thanks Addison! 
This was link[1] was perfect, just copying a file, changing one number and updating some table in mysql.

[1]
https://www.atwix.com/magento/custom-order-and-customer-numbers/
